vignettes/quick_start.Rmd

```diff
@@ -0,0 +1,97 @@
+---
+title: "Getting Started"
+date: "2017-08-21"
+output: rmarkdown::html_vignette
+vignette: >
+  %\VignetteIndexEntry{Asychronous Programming}
+  %\VignetteEngine{knitr::rmarkdown}
+  %\VignetteEncoding{UTF-8}
+---
+
+## A (Very) Quick Start
+
+To begin, make sure `civis` is installed and your
+[API key is in your R environment](https://civisanalytics.github.io/civis-r/#installation).
+You can quickly test that `civis` is working by invoking
+
+```{r, eval=FALSE}
+name <- civis::users_list_me()$name
+paste(name, "is really awesome!")
+```
+
+If `civis` is working, you'll see a friendly message.  Otherwise, you might see an
+error like
+
+```{r, eval=FALSE}
+Error in loadNamespace(name) : there is no package called 'civis'
+```
```

Comment:
Should we mention the warning/error that turns up when `CIVIS_API_KEY` is missing?